### DISCUSSION TITLE
Can we include more custom fields in the registration form?

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Lern/Community#82] [Query]
Hello @reshmi-nair, @rhwarrier

Can we include more custom fields in the sunbird-ed registration form?

CC - @Renuka-S, @Krishnaj20

**Comment by reshmi-nair** [comment]
@komalm Yes, it can be done. But changes are required in registration form(configured in Keycloak), Sunbird-Ed Portal backend and Sunbird-Lern UserOrg service APIs
